app-android-xamarin/Moodis/Moodis/Feature/Menu/MenuActivity.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+using Android.App;
+using Android.Content;
+using Android.Graphics;
+using Android.OS;
+using Android.Support.V7.App;
+using Android.Util;
+using Android.Widget;
+using Java.Lang;
+using Moodis.Feature.CameraFeature;
+using Moodis.Ui;
+
+namespace Moodis.Feature.Menu
+{
+    [Activity(Label = "Menu Activity")]
+    public class MenuActivity : AppCompatActivity
```

Comment:
Just a thought, but maybe it would be reasonable to use Main Activity for Main Menu things?

Reply:
What would we gain from using it like this ?

Reply:
Main activity won't be empty :D